packages/functional-tests/pages/login.ts

```diff
@@ -378,22 +386,58 @@ export class LoginPage extends BaseLayout {
     return this.page.locator(selectors.SUBMIT).click();
   }
 
-  async waitForCWTSHeader() {
-    await this.page.waitForSelector(selectors.CWTS_HEADER, {
-      timeout: 100,
+  async waitForCWTSEngineHeader() {
+    await this.page.waitForSelector(selectors.CWTS_ENGINE_HEADER, {
+      timeout: 2000,
     });
   }
 
-  async waitForCWTSEngineBookmarks() {
-    await this.page.waitForSelector(selectors.CWTS_ENGINE_BOOKMARKS, {
-      timeout: 100,
-    });
+  async isCWTSPageHeader() {
+    return this.page.locator(selectors.CWTS_PAGE_HEADER).isVisible();
   }
 
-  async waitForCWTSEngineHistory() {
-    await this.page.waitForSelector(selectors.CWTS_ENGINE_HISTORY, {
-      timeout: 100,
-    });
+  async isCWTSEngineCreditCards() {
+    return this.page.locator(selectors.CWTS_ENGINE_CREDITCARDS).isVisible();
+  }
+
+  async isCWTSEngineBookmarks() {
+    return this.page.locator(selectors.CWTS_ENGINE_BOOKMARKS).isVisible();
+  }
+
+  async isCWTSEngineHistory() {
+    return this.page.locator(selectors.CWTS_ENGINE_HISTORY).isVisible();
+  }
+
+  async isCWTSEnginePassword() {
+    return this.page.locator(selectors.CWTS_ENGINE_PASSWORDS).isVisible();
+  }
```

Comment:
nit: rename method to `*Passwords()` plural to match `selectors.CWTS_ENGINE_PASSWORDS` name and be consistent w/ `uncheckCWTSEnginePasswords()` function name below.

```suggestion
  async isCWTSEnginePasswords() {
    return this.page.locator(selectors.CWTS_ENGINE_PASSWORDS).isVisible();
  }
```